eras/conway/impl/src/Cardano/Ledger/Conway/Governance.hs

```diff
@@ -888,7 +889,8 @@ computeDrepDistr ::
   Map (Credential 'DRepRole c) (DRepState c) ->
   Map (Credential 'Staking c) (CompactForm Coin) ->
   Map (DRep c) (CompactForm Coin)
-computeDrepDistr um regDreps stakeDistr = Map.foldlWithKey' (accumDRepDistr um regDreps) Map.empty stakeDistr
+computeDrepDistr um regDreps stakeDistr =
+  Map.foldlWithKey' (accumDRepDistr um regDreps) Map.empty stakeDistr
```

Comment:
I was wondering if other call sites in the same module that use `Map.foldlWithKey' (accumDRepDistr um regDreps) Map.empty stakeDistr` could also just call `computeDrepDistr`, whether that will be to our benefit.

Reply:
There are no other callsites in the module that use :wink: 
```haskell
Map.foldlWithKey' (accumDRepDistr um regDreps) Map.empty stakeDistr
```